fedimint-testing/src/ln/real.rs

```diff
@@ -415,12 +420,14 @@ impl LdkLightningTest {
         bitcoin: Arc<dyn BitcoinTest>,
     ) -> Result<LdkLightningTest, LightningRpcError> {
         let mut builder = Builder::new();
-        builder.set_network(Network::Regtest);
+        builder.set_network(ldk_node::Network::Regtest);
         let unused_port = find_unused_port().expect("Could not find an unused port for LdkNode");
-        builder.set_listening_address(
-            NetAddress::from_str(format!("0.0.0.0:{unused_port}").as_str())
-                .expect("Couldnt parse listening address"),
-        );
+        builder
+            .set_listening_addresses(vec![SocketAddress::from_str(
+                format!("0.0.0.0:{unused_port}").as_str(),
+            )
+            .expect("Couldnt parse listening address")])
+            .expect("Couldnt set listening address");
         builder.set_storage_dir_path(db_path.to_string_lossy().to_string());
         let esplora_port = std::env::var("FM_PORT_ESPLORA").unwrap_or(String::from("50002"));
         builder.set_esplora_server(format!("http://127.0.0.1:{esplora_port}"));
```

Comment:
Odd - when running the real tests locally for bitcoind, which now covers fedimint-ln-gateway tests, this passes. Out of scope for this PR, but perhaps we should run the ln-gateway backend tests against esplora instead of bitcoind?